### ISSUE TITLE
[Feature]: Add o_index to core_fields_configuration

### ISSUE THREAD

**Issue by valentin-** [pimcore/advanced-object-search#134]
### Feature description

Hi, I'd like to add the `o_index` field to the index configuration to provide sorting based on the backend list.

I've added the field in the config, unfortunately it won't get added to the list automatically https://github.com/pimcore/advanced-object-search/blob/4.x/src/Service.php#L185.

```
core_fields_configuration:
        -   field: o_index
            type: long
            title: Index
            fieldDefinition: Pimcore\Model\DataObject\ClassDefinition\Data\Numeric
```

It would be great if theres a possibility to add more core fields, I can also create a PR to add the single field definition if you don't mind adding it to the core.

Thanks,
Valentin

**Comment by fashxp** [comment]
looks like somehow like a bug to me. we definitely should finish the possibility to add more core fields. 
any PR is welcome ;-)

**Comment by dvesh3** [comment]
Fixed by #135
